src/Models/ItemModel.vala

```diff
@@ -0,0 +1,24 @@
+/*
+* Copyright (c) 2019 Alecaddd (https://alecaddd.com)
+*
+* This file is part of Akira.
+*
+* Akira is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+* Akira is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+* You should have received a copy of the GNU General Public License
+* along with Akira. If not, see <https://www.gnu.org/licenses/>.
+*
+* Authored by: Alessandro "alecaddd" Castellani <castellani.ale@gmail.com>
+*/
+
+public abstract class Akira.Models.ItemModel : GLib.Object {
+    public Lib.Models.CanvasItem item { get; construct; }
+    public Akira.Models.ListModel list_model { get; set; }
+    public Akira.Utils.BlendingMode blending_mode { get; set; }
```

Comment:
Why do we need to have the blending model in here? 

I think this is a leftover, this should go in the `FillItemModel` and/or in the `BorderItemModel`, since it's not a general attribute of the model. 

Do you agree?

Reply:
Indeed. I left it there because I was going to deal with it once adding the dashed style for the border. I guess I can change it now.